### PR TITLE
AR Pointer event simulation on screen target ray mode

### PR DESCRIPTION
**Pull request by RaananW**
'screen' target ray mode (mainly mobile) will now have emulated pointer events that will allow interaction with the scene

### CODE REVIEW

**Comment by deltakosh** [comment]
We still have a doc issue ;(

**Comment by RaananW** [comment]
It is all there, but it seems like the doc-check doesnt like readonly definitions in the constructor.
I will remove the readonly

**Comment by deltakosh** [comment]
Still ;D

**Comment by deltakosh** [comment]
image

**Comment by RaananW** [comment]
I keep on searching, but it is there - 

image

checking further :-)

File: src/Cameras/XR/webXREnterExitUI.ts [hunk 1]
```diff
@@ -117,7 +117,12 @@ export class WebXREnterExitUI implements IDisposable {
         });
     }
 
-    private constructor(private scene: Scene, options: WebXREnterExitUIOptions) {
+    /**
+     *
+     * @param scene babylon scene object to use
+     * @param options (read-only) version of the options passed to this UI
+     */
+    private constructor(private scene: Scene, public options: WebXREnterExitUIOptions) {
```
Comment (deltakosh) [inline src/Cameras/XR/webXREnterExitUI.ts]: correct one is:
/**
     * BLA BLA BLA
     * @param scene babylon scene object to use
     * @param options (read-only) version of the options passed to this UI
     */
    private constructor(private scene: Scene, 
      /** BBLA BLA BLA */
      public options: WebXREnterExitUIOptions) {

File: src/Cameras/XR/webXRSessionManager.ts [hunk 2]
```diff
@@ -62,7 +66,7 @@ export class WebXRSessionManager implements IDisposable {
      * Constructs a WebXRSessionManager, this must be initialized within a user action before usage
      * @param scene The scene which the session should be created for
      */
-    constructor(private scene: Scene) {
+    constructor(public scene: Scene) {
```
Comment (deltakosh) [inline src/Cameras/XR/webXRSessionManager.ts]: same here the scene parameter needs a comment

**Comment by deltakosh** [comment]
Yeahhh! ;)

**Pull request merged**